### PR TITLE
Idle run fix

### PR DESCRIPTION
**Pull request by lfdversluis**
fixes **1** bug

### CODE REVIEW

**Comment by lfdversluis** [comment]
@devos50 can you review this?

**Comment by devos50** [comment]
Looks good! I don't see anything special.

**Comment by lfdversluis** [comment]
@whirm ready for your review!

File: experiments/tribler_idle_run/tribler_idle_run.py [hunk 1]
```diff
@@ -37,36 +37,43 @@
...
 import os
+from twisted.internet import reactor
 
 from gumby.instrumentation import init_instrumentation
 
-os.chdir(os.path.abspath('./tribler'))
-sys.path.append('.')
 
-from Tribler.Test.test_as_server import TestGuiAsServer
+sys.path.append('./tribler')
```
Comment (whirm) [inline experiments/tribler_idle_run/tribler_idle_run.py]: Use absolute paths or importing will break if CWD changes at runtime

**Comment by whirm** [comment]
Looks fine, fix/squash and I'll merge

**Comment by lfdversluis** [comment]
@whirm I've wrapped the two lines in  `os.path.abspath()`.

**Pull request merged**